Makefile

```diff
@@ -47,3 +48,6 @@ push: ./manifest-tool $(addprefix push-,$(ALL_ARCH))
 ./manifest-tool:
 	curl -sSL https://github.com/estesp/manifest-tool/releases/download/v0.5.0/manifest-tool-linux-amd64 > manifest-tool
 	chmod +x manifest-tool
+
+vendor: glide.lock
+	glide install -v
```

Comment:
This assumes glide on host. I can install it for sure, but have everything I need in docker containers as a rule of thumb.